configs/fedora.toml

```diff
@@ -38,6 +44,15 @@ queue = "00000000-0000-0000-0000-000000000000"
 exchange = "amq.topic"
 routing_keys = ["#"]  # Set this to the specific topics you are interested in.
 
+# To receive messages from the fedmsg->fedora-messaging bridge,
+# duplicate the binding but for the 'zmq.topic' exchange. It's safe to
+# bind to all the routing_keys you are interested in on both exchanges
+# so you keep working when a publisher is ported to fedora-messaging.
```

Comment:
Since this is only relevant to internal /pubsub users, maybe at the bottom add
```
# This is only required for users of the "/pubsub" exchange, not those using the 
# "/public_pubsub" virtual host and the "fedora" user
```

Reply:
oh, if that's the case, I think we should leave it out; I wasn't intending to include stuff that was specific to that case here. It would be good to cover that kinda stuff somewhere, but I'm not sure where.